asv_bench/benchmarks/benchmarks.py

```diff
@@ -1184,4 +1184,80 @@ def time_series_indices(self, shape):
         self.series.groupby(self.series).indices
 
 
+class TimeRepr:
+
+    params = [get_benchmark_shapes("TimeRepr")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        self.df = IMPL.DataFrame(np.random.randn(*shape))
+        execute(self.df)
+
+    # returns a  string thus not calling execute
+    def time_repr(self, shape):
+        repr(self.df)
+
+
+class TimeMaskBool:
+
+    params = [get_benchmark_shapes("TimeMaskBool")]
+    param_names = ["shape"]
+
+    def setup(self, shape):
+        df = IMPL.DataFrame(np.random.randn(*shape))
+        df = df.where(df > 0)
+        self.bools = df > 0
+        self.mask = IMPL.isnull(df)
+        execute(self.bools), execute(self.mask)
+
+    def time_frame_mask(self, shape):
+        execute(self.bools.astype(float).mask(self.mask))
```

Comment:
Why not just like that?
```suggestion
        self.df = IMPL.DataFrame(np.random.randn(*shape))
        self.mask = df < 0
        execute(self.df), execute(self.mask)

    def time_frame_mask(self, shape):
        execute(self.df.mask(self.mask))
```

Reply:
this seems better.